src/App.tsx

```diff
@@ -1,14 +1,59 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { FilterStatusType, Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+function filterFunction(
+  items: Todo[],
+  str: string,
+  filterStates: FilterStatusType,
+) {
+  let itemsInUse = [...items];
+
+  itemsInUse = itemsInUse.filter(todo =>
+    todo.title.toLowerCase().includes(str.toLowerCase().trim()),
+  );
```

Comment:
let's get rid of using let

const x = [...items].filter(todo =>
    todo.title.toLowerCase().includes(str.toLowerCase().trim()),
  );
  
  
  return x.filter (item => {
    switch (filterStates) { ...

Reply:
> let's get rid of using let
> 
> const x = [...items].filter(todo => todo.title.toLowerCase().includes(str.toLowerCase().trim()), );
> 
> return x.filter (item => { switch (filterStates) { ...

Not fixed